lib/Validator/GeographyValidator.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Copyright (C) 2016 Derek J. Lambert
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+namespace CrEOF\Geo\Obj\Validator;
+
+use CrEOF\Geo\Obj\Exception\RangeException;
+
+/**
+ * Class GeographyValidator
+ *
+ * @author  Derek J. Lambert <dlambert@dereklambert.com>
+ * @license http://dlambert.mit-license.org MIT
+ */
+class GeographyValidator implements ValidatorInterface
+{
+    const CRITERIA_LONGITUDE_FIRST = 0;
+    const CRITERIA_LATITUDE_FIRST  = 1;
+
+    /**
+     * @var int
+     */
+    private $order;
+
+    /**
+     * GeographyValidator constructor
+     *
+     * @param int $order
+     */
+    public function __construct($order)
+    {
+        $this->order = $order;
+    }
+
+    /**
+     * @param array $value
+     *
+     * @throws RangeException
+     */
+    public function validate(array $value)
+    {
+        if ('point' !== $value['type']) {
```

Comment:
1. Currently the validator stack for a type is being called from ```Object::__construct()```, and a check for the ```value``` key is done beforehand. I considered adding another check in the validators but it seemed redundant. The validators will be called for each new object, so was trying to avoid unnecessary operations. I do see a case for it if someone was using the classes in an unintended way. I don't have a strong opinion either way.
2. Agreed, that would be cleaner and definitively indicate if validation was performed.